website/docs/core-concepts/stacks/yaml-functions/store.mdx

```diff
@@ -60,9 +60,18 @@ components:
     my_lambda_component:
       vars:
         vpc_config:
+          # AWS SSM Parameter Store example
           security_group_id: !store security-group/lambda id
           security_group_id2: !store security-group/lambda2 {{ .stack }} id
           security_group_id3: !store security-group/lambda3 {{ .atmos_stack }} id
+
+          # Google Secret Manager examples
+          db_password: !store database password
+          api_key: !store api credentials | default "default-key"
+          # Cross-stack example with GSM
+          shared_secret: !store secrets {{ .stack }} api_token
+          # Using template for dynamic stack name with GSM
+          config_value: !store config {{ printf "%s-%s" .vars.environment .vars.stage }} setting
```

Comment:
I don't think we need GSM specific examples of using the `!store` function because it works identically no matter what the backing store.